client/components/PlayerHand.js

```diff
@@ -106,6 +108,58 @@ class PlayerHand extends React.Component {
       this.setState({
         updating: false
       })
+    } else if (
+      this.props.me.hand &&
+      this.props.me.hand.length <= 1 &&
+      this.props.me.number === 1 &&
+      this.props.age === 3 &&
+      !this.state.updating
+    ) {
+      console.log('game over')
+      this.setState({
+        updating: true
+      })
+      let playersToUpdate = []
+      let updatedMilitary = []
+      await this.props.firestore
```

Comment:
Not sure we need await when we already have .then() but it doesn't break anything so we can worry about removing these down the road